bulbs/videos/static/videos/js/series/series-meta/series-meta.js

```diff
@@ -0,0 +1,29 @@
+SeriesMeta = function() {
+  this.$seriesGrid = $('#series-video-list');
+  this.videohubBase = this.$seriesGrid.data('videohub-base');
+  this.seriesSlug = this.$seriesGrid.data('series-slug');
+  this.$seriesTitle = $('#series-title');
+  this.source = this.videohubBase + '/series/' + this.seriesSlug + '.json';
+  this.fetchSeriesMeta();
+};
+
+SeriesMeta.prototype.fetchSeriesMeta = function() {
+  $.getJSON(this.source, this.seriesMetaFetched.bind(this));
+};
+
+SeriesMeta.prototype.seriesMetaFetched = function(data) {
+  $('#series-description').html(data.series_description);
+  this.$seriesTitle.html(data.series_name);
+  /* Commenting this out until we resolve logo crop issue
```

Comment:
This requires that our design team create standard crops for all series logos and re-upload them to Onions Studios. Kelly is aware of the issue and we chose to launch without the logo